### PR TITLE
starting files update

### PR DESCRIPTION
**Pull request by ashuvssut**
changes description:

- settings.py: new changes solves a error that occurred when running django server. some other minor changes are also there

- Readme.md: updated 'environment setup' section made for both --Unix based OSes and Windows

### CODE REVIEW

File: cms_backend/settings.py [hunk 1]
```diff
@@ -32,13 +32,12 @@
 
 INSTALLED_APPS = [
     'django.contrib.admin',
-    'users.apps.UsersConfig'
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'django_graphene'
```
Comment (partha2000) [inline cms_backend/settings.py:41]: Why did you delete graphene?

File: cms_backend/settings.py [hunk 2]
```diff
@@ -32,13 +32,12 @@
 
 INSTALLED_APPS = [
     'django.contrib.admin',
-    'users.apps.UsersConfig'
```
Comment (partha2000) [inline cms_backend/settings.py:35]: Dont remove `users.apps.UsersConfig` as of now.

**Review by partha2000 — CHANGES_REQUESTED** [review]
The addition of environment setup for windows systems is appreciated. However leave the environment setup for Linux systems as it is and add another one below for windows. Format it similar to the original style.

File: cms_backend/settings.py (hunk 1 above)
Comment (ashuvssut) [reply cms_backend/settings.py:41]: @partha2000  the command to run django runsrever failed

see the attached SS  below:
image

File: cms_backend/settings.py (hunk 2 above)
Comment (ashuvssut) [reply cms_backend/settings.py:35]: but, the command to run django runsrever failed

see the attached SS  below:
image

Comment (partha2000) [reply cms_backend/settings.py:35]: The problem is you haven't registered the `users` app under `INSTALLED_APPS` in `settings.py`. You'll have to register each app you create inside `INSTALLED_APPS`. Add `users.apps.UsersConfig` inside installed apps.

File: cms_backend/settings.py (hunk 1 above)
Comment (partha2000) [reply cms_backend/settings.py:41]: The `requirements.txt` was most probably missing graphene. It has been updated. Just do this to solve this error: `pip install -r requirements.txt`.

**Comment by partha2000** [comment]
Update the PR description as well according to the new changes.

**Review by partha2000 — DISMISSED** [review]

File: cms_backend/settings.py (hunk 1 above)
Comment (ashuvssut) [reply cms_backend/settings.py:41]: ok understood. i will pull n update my pr

File: cms_backend/settings.py (hunk 2 above)
Comment (ashuvssut) [reply cms_backend/settings.py:35]: ohk got it
